charts/flyte-core/values.yaml

```diff
@@ -16,7 +16,7 @@ flyteadmin:
   image:
     # -- Docker image for Flyteadmin deployment
     repository: cr.flyte.org/flyteorg/flyteadmin # FLYTEADMIN_IMAGE
-    tag: v1.1.29-hotfix # FLYTEADMIN_TAG
+    tag: v1.1.30 # FLYTEADMIN_TAG
```

Comment:
We need to keep the hotfix image until the fix is merged to master

Reply:
reverted